tests/playwright/config/global-setup.js

```diff
@@ -7,8 +7,9 @@ module.exports = async ( config ) => {
 	await page.goto( `${ config.projects[ 0 ].use.baseURL }/wp-admin` );
 
 	await page.waitForSelector( 'text=Log In' );
```

Comment:
Isn't `input[name="wp-submit"]` more appropriate?

Reply:
Also in line 12

Reply:
@joshmarom I didn't write those lines, but the E2E tests purpose is to reflect the way a user would of use the application, so I guess in this matter "Log In" is more appropriate.

Reply:
I still think `input[name="wp-submit"]` or `#wp-submit` would be a more resilient choice

see https://github.com/woocommerce/wp-e2e-page-objects/blob/master/src/components/wp-admin/component-login-form.js#L13-L16

Anyway, it's just a recommendation. :)